public/register.js

```diff
@@ -0,0 +1,85 @@
+import {
+    inputEnabled,
+    setDiv,
+    message,
+    token,
+    enableInput,
+    setToken,
+  } from "./index.js";
+  import { showLoginRegister } from "./loginRegister.js";
+  import { showJobs } from "./jobs.js";
```

Comment:
Should use showStudents
```suggestion
  import { showStudents } from "./student.js";
```